doc/sphinx/language/extensions/arguments-command.rst

```diff
@@ -0,0 +1,445 @@
+.. _ArgumentsCommand:
+
+Setting properties of a function's arguments
+++++++++++++++++++++++++++++++++++++++++++++
+
+.. cmd:: Arguments @smart_qualid {* @arg_specs } {* , {* @implicits_alt } } {? : {+, @args_modifier } }
+   :name: Arguments
+
+   .. insertprodn smart_qualid args_modifier
+
+   .. prodn::
+      smart_qualid ::= @qualid
+      | @by_notation
+      by_notation ::= @string {? % @scope_key }
+      argument_spec ::= {? ! } @name {? @scope }
+      arg_specs ::= @argument_spec
+      | /
+      | &
+      | ( {+ @argument_spec } ) {? @scope }
+      | [ {+ @argument_spec } ] {? @scope }
+      | %{ {+ @argument_spec } %} {? @scope }
+      implicits_alt ::= @name
+      | [ {+ @name } ]
+      | %{ {+ @name } %}
+      args_modifier ::= simpl nomatch
+      | simpl never
+      | default implicits
+      | clear implicits
+      | clear scopes
+      | clear bidirectionality hint
+      | rename
+      | assert
+      | extra scopes
+      | clear scopes and implicits
+      | clear implicits and scopes
+
+   .. todo does the command really need smart_qualid or just qualid?
+      in fact, "Arguments Nat.add%nat_scope {n}%bool_scope m%nat : assert." gives a syntax error on the %
```

Comment:
`smart_qualid` is not about `%scope`, and I make use of the "smart" feature in fiat-crypto.  It lets you do things like
```coq
Notation foo := Nat.add.
Arguments foo ....
```
I think it also lets you do this if you fill in some initial arguments in the alias.

Reply:
Yeah, I misread smart_qualid.  I removed the todo.